troi/content_resolver/database.py

```diff
@@ -353,7 +350,10 @@ def update_status(self, statusdata):
             Update status counter and display matching progress
         """
         self.counters.status[statusdata.status] += 1
-        if not self.quiet:
+
+        if self.quiet:
```

Comment:
In future, I think it would be better to configure logging level based on the quiet flag in troi's entry points. Also, configure tqdm with it if possible. But should be fine for now.